### DISCUSSION TITLE
How to setup MonicaHQ

### DISCUSSION THREAD

**Discussion by Turcu09** [monicahq/monica#7361] [Q&A]
Hello dear people,

Looking for some assistance in regards to setting up MonicaHQ and deciding whether to choose V4 or V5. 
While not being a complete noob, I still cannot understand how to properly set it up and start using it. 
From my research so far, this would be the best option in regards to a personal CRM and therefore my reach to the community. 
While I understand that I cannot expect for someone to take their time to show me the complete process live, I would really appreciate a step-by-step guide on how to make something out of it using Docker Desktop. 

Any help is much appreciated.

**Comment by Szeraax** [comment]
Do you have familiarity with docker itself? I ask because there are lots of youtube and other online resources that can help you learn how to host things with docker. And once you understand docker, doing Monica vs Nginx vs many others is pretty trivial. 

Having said that, do you know that Docker Desktop services CANNOT be accessed by ANY computer besides the local host. It is not your desired long term path if you want to be able to access monica from your desktop and your smartphone. Or from over the internet. 

I'd suggest that you focus on learning docker (such as with a simple nginx container first) and then focus on learning about volumes (probably bind mounts) so that you aren't going to put in all your data and then accidentally delete it all the first time you try to prune some space.

**Reply by Turcu09** [reply]
Thank you for your reply.
I've completely paused the process due to lack of understanding and rushing towards setting up Monica. 
I will try to document more in regards to Docker and start setting everything up.

Once again, appreciate the reply.
